src/components/DelegationNode/DelegationNode.tsx

```diff
@@ -276,6 +276,13 @@ class DelegationNode extends React.Component<Props, State> {
 
     const hashes = await delegation.getAttestationHashes()
 
+    if (hashes.length < 1) {
+      notify(
+        <span>No attestations associated with this Delegation</span>,
+        false
+      )
+    }
+
```

Comment:
```suggestion
```

Reply:
pulled this out to https://github.com/KILTprotocol/demo-client/pull/345, will remove here now